### PR TITLE
[IMP] Website: Unsplash documentation updated

### PR DESCRIPTION
**Pull request by Donapi**
[taskid: 3498880](https://www.odoo.com/web#id=3498880&cids=1&model=project.task&view_type=form)

### CODE REVIEW

**Review by auva-odoo — CHANGES_REQUESTED** [review]
@Donapi Thank you for your work! Since I will be gone when you see this and we therefore won't be able to discuss this IRL until I get back, I wrote down my general thoughts so you're not completely stuck while I'm gone. I also *tried* to explain my suggestions for the same reason (these might be "to-the-point" because I'm a bit short on time before my holiday). I hope this all makes sense to you :-) 

1) I think there are some issues with your images:
- It seems you deleted the "unsplash" folder (with your images) in /home/odoo/documentation/content/applications/websites/website/configuration, which I think is the reason why the PR failed the ci/documentation check.
- After discussing this with jcs when I took over the website documentation, we decided that images would not be centered throughout the website doc and instead aligned on the left (you can just remove  `:align: center` for all images; images are aligned on the left by default)
- I think you forgot to add the alt text for your images ( :alt: )

2) I would remove all of the subtitles and instead have one big list (a numbered list) of steps (I also think we could group some of those steps together so we don't have as many - see my suggestions) like we did for the Website analytics page (Plausible and Google Analytics) https://www.odoo.com/documentation/16.0/applications/websites/website/reporting/analytics.html 
I feel like all of the steps actually go together, especially in the first two sections; in the second section, you re-explain how to access the applications details page, when we were exactly there at the previous step, in the previous section. Removing the sub-titles would allow you to remove the confusion and unnecessary steps, IMO.

File: content/applications/websites/website/configuration/unsplash.rst [hunk 1]
```diff
@@ -45,22 +37,29 @@ Generate an Unsplash access key for non-Odoo Online users
...
-
-.. image:: unsplash/select_app.png
-    :align: center
+  Unsplash application.
 
-- You will be redirected to your application details page. The **application ID** will be visible in
-  your browser's URL. The URL should be something like
-  ``https://unsplash.com/oauth/applications/<application_id>``
+- You are redirected to your application details page. The **application ID** is visible in your
+  browser's URL.
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/websites/website/configuration/unsplash.rst:44]: yes, it is, not sure why you can't see the screenshot...

File: content/applications/websites/website/configuration/unsplash.rst [hunk 2]
```diff
@@ -1,66 +1,45 @@
...
 
-.. image:: unsplash/create_app.png
-   :align: center
+#. Access your `applications dashboard <https://unsplash.com/oauth/applications>`_, click
+   :guilabel:`New Application`, select all checkboxes, and click :guilabel:`Accept terms`.
 
-- Accept the conditions and click on **Accept terms**.
+#. In the pop-up window that opens, enter your :guilabel:`Application Name`, starting with the
+   prefix `Odoo:` (e.g., `Odoo: connection`) so Unsplash recognizes it as an Odoo instance. Then,
+   add a :guilabel:`Description`, and click :guilabel:`Create application`.
```
Comment (auva-odoo) [inline content/applications/websites/website/configuration/unsplash.rst:18]:
```suggestion
   prefix `Odoo:` (e.g., `Odoo: connection`), so Unsplash recognizes it as an Odoo instance. Then,
   add a :guilabel:`Description` and click :guilabel:`Create application`.
```

File: content/applications/websites/website/configuration/unsplash.rst [hunk 3]
```diff
@@ -1,66 +1,45 @@
...
-.. image:: unsplash/accept_terms.png
-   :align: center
+#. On the application details page, scroll down to the :guilabel:`Keys` section and copy the
+   :guilabel:`Access key`.
 
-- You will be prompted to insert an **Application name** and a **Description**. Please prefix your
-  application name by "**Odoo:** " so that Unsplash can recognize it as an Odoo instance. Once done,
-  click on **Create application**.
+#. To **generate an Unsplash application ID** go to your `applications dashboard <https://unsplash.com/oauth/applications>`_
+   and click on the application you just created.
```
Comment (auva-odoo) [inline content/applications/websites/website/configuration/unsplash.rst:24]:
```suggestion
#. To **generate an Unsplash application ID**, go to your `applications dashboard <https://unsplash.com/oauth/applications>`_
   and click on the application you just created.
```

**Review by auva-odoo — CHANGES_REQUESTED** [review]
Hey @Donapi you did a great job there :) I know it seems like I still have a lot of comments but really it's just a few minor changes (commas, image alignment) and 1 suggestion :) let me know if you want to talk about it :)

File: content/applications/websites/website/configuration/unsplash.rst [hunk 4]
```diff
@@ -1,66 +1,45 @@
...
+   :alt: Identify your app idd url.
 
-.. image:: unsplash/access_key.png
-   :align: center
-
-.. warning::
-   **As a non-Odoo Online user**, you won't be able to register for a production Unsplash key and
-   will be limited to your test key that has a restriction of 50 Unsplash requests per hour.
+7. In Odoo, go to :menuselection:`General Settings --> Integrations: Unsplash Image Library`, and
+   enter your :guilabel:`Access Key` and :guilabel:`Application ID`.
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/websites/website/configuration/unsplash.rst:32]: I think it was ok as it was but I updated it...

**Review by auva-odoo — APPROVED** [review]

**Comment by xpl-odoo** [comment]
@robodoo r+

**Pull request closed without merging**